### DISCUSSION TITLE
2D Taylor-Green-Vortex

### DISCUSSION THREAD

**Discussion by MarcoArtiano** [xcompact3d/Incompact3d#237] [Q&A] [answered]
Hello,

I would like to run a 2D Taylor-Green-Vortex for a spatial and temporal accuracy analysis, but when I set up the same conditions as in input_2d.i3d, I get the following error:

''

 TGV 2D:  F
 ===========================================================
 In auto-tuning mode......
 factors:            1           2           3           6
 p_row x p_col           2           3
 2DECOMP&FFT ERROR - errorcode:            6
 ERROR MESSAGE: Invalid 2D processor grid. Make sure that min(nx,ny) >= p_row and min(ny,nz) >= p_col

''
how can I fix this?

Thank you in advance!

**Comment by slaizet** [comment, marked as answer]
Hi,
So I just tried to run the 2D TGV case using https://github.com/xcompact3d/Incompact3d/blob/master/examples/Taylor-Green-Vortex/input_2D.i3d and it is working fine with 6 and 8 MPI processes (with GNU Fortran 9.4).
Did you make changes to the input file?
Best,
Sylvain

**Reply by MarcoArtiano** [reply]
Thanks, I made few changes, and after restoring it to the original version it is working again.
